docs/modpacks/playing_modpacks.md

```diff
@@ -61,3 +61,22 @@ Click the "Add a new Minecraft Core or Mod Pack" tile in the Start Menu, then cl
 ![Add a new Minecraft Core or Mod Pack Tile](../../static/img/tutorial/BakaXLAddCoreTile.png)
 
 Click "Continue" and drag the modpack file into BakaXL.
+
+### HMCL
+
+HMCL is a cross-platform Minecraft launcher which supports Mod Management, Game Customizing, Auto Installing (Forge, Fabric, Quilt, LiteLoader and OptiFine), Modpack Creating, UI Customization, and more.
+![HMCL](../../static/img/tutorial/Hmcl.png)
```

Comment:
This document isn't an ad for HMCL, it's a guide on how to install modpacks using these tools. Please rephrase this so that it reads less like an ad and more like the other sections. This image is completely unnecessary.

Reply:
Thank you for pointing that out, it has been corrected.